### DISCUSSION TITLE
Unable to read midi file

### DISCUSSION THREAD

**Discussion by NoteBlockMR** [otomad/OtomadHelper#71] [Q&A] [answered]
How do I solve this problem?

The test version is Vegas 16
![Screenshot_2](https://github.com/otomad/OtomadHelper/assets/170589000/5fb42899-ef0f-4766-b2d5-562a14d5356d)

**Comment by Evauation** [comment, marked as answer]
https://github.com/otomad/OtomadHelper/issues/25 - Velocity Error!

    System.FormatException: Got an off without an on 0 NoteOn

# English:

Your MIDI file has a note that has a velocity of 0. Check to make sure there isn't any notes that have a velocity value of 0 (aka notes that have no volume).

You can generally fix this by reimporting the MIDI file back into FL Studio and exporting it again or going back to the original midi and removing the notes with no velocity.

# Korean

MIDI 파일에 속도 값이 0인 노트가 있습니다. 속도 값이 0인 노트(일명 볼륨이 없는 노트)가 없는지 확인합니다.

일반적으로 MIDI 파일을 FL Studio로 다시 불러온 다음 다시 내보내거나 원본 미디로 돌아가서 벨로시티가 없는 노트를 제거하면 이 문제를 해결할 수 있습니다.
